docs/data/material/customization/how-to-customize/how-to-customize.md

```diff
@@ -96,15 +89,17 @@ Target the `.Mui-selected` global class name to customize the special state of t
 <MenuItem selected className="MenuItem">
 ```
 
+If you'd like to learn more about CSS specificity, we recommend checking [MDN's guide](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) about it.
+
 #### Why do I need to increase specificity to override one component state?
 
 By design, the CSS specification makes the pseudo-classes increase the specificity.
-For consistency with native elements, MUI increases the specificity of its custom state classes.
-This has one important advantage, it allows you to cherry-pick the state you want to customize.
+For consistency with native elements, Material UI increases the specificity of its custom state classes.
+This has one important advantage: it allows you to cherry-pick the state you want to customize.
```

Comment:
I don't think I fully understand the answer to this question. Here's what I gather:

- pseudo-classes have a high level of specificity
- MUI's state classes also have a high level of specificity - possibly more than CSS pseudo-classes?
- this is a good thing because it makes it simple to target individual components using these classes

Is that correct?

Reply:
- pseudo-classes have a high level of specificity ✅ 
- MUI's state classes also have a high level of specificity ~possibly more than CSS pseudo-classes~? they have the same as pseudo states, see https://codepen.io/mnajdova/pen/mdpwxyY
- this is a good thing because it makes it simple to target individual ~components~ component's state using these classes